opentelemetry-sdk/src/opentelemetry/sdk/metrics/__init__.py

```diff
@@ -260,5 +270,27 @@ def create_metric(
             monotonic=monotonic,
         )
 
+    def get_label_set(self, labels: Dict[str, str]):
+        """See `opentelemetry.metrics.Meter.create_metric`.
+
+        This implementation encodes the labels to use as a map key.
+
+        Args:
+            labels: The dictionary of label keys to label values.
+        """
+        if len(labels) == 0:
+            return EMPTY_LABEL_SET
+        sorted_labels = OrderedDict(sorted(labels.items()))
+        # Uses statsd encoding for labels
```

Comment:
Good question! I am following the [Go implementation](https://github.com/open-telemetry/opentelemetry-go/blob/master/sdk/metric/sdk.go#L235) which apparently uses what statsd is using.

Reply:
The type of encoding is used for potential optimizations that can occur for corresponding exporters that might be used. From the discussion in the metrics-spec gitter, the statsd encoding is a default implementation that is used. Until we implement exporters, the benefits of this won't be that clear. As well in the Go SDK implementation, there is a concept of a `LabelEncoder` , which allows plugging in custom encoding for labelsets. Take a look at `labelencoder.go` in this [PR](https://github.com/open-telemetry/opentelemetry-go/pull/265/files)

Reply:
It seems like it would be much simpler to make `LabelSet` hashable and just return the `LabelSet` itself without doing any encoding here. The meter shouldn't have an opinion on labelset encoding, that's a problem for exporters.

If there's no benefit of doing the encoding up-front until we have a statsd exporter, I think we should wait to add this until we add the exporter.